### ISSUE TITLE
[mod_verto] loop cicle and not accepting new connections after WRITE RETURNED ERROR -1

### ISSUE THREAD

**Issue by wmasilva** [signalwire/freeswitch#593]
hi,
I'm having a lock issue with mod_verto,  i got an ALERT message:

2020-04-29 10:21:58.016189 [ALERT] mod_verto.c:622 WRITE RETURNED ERROR -5 
2020-04-29 10:21:58.016189 [ALERT] mod_verto.c:622 WRITE RETURNED ERROR -1 
2020-04-29 10:21:58.016189 [ALERT] mod_verto.c:622 WRITE RETURNED ERROR -1 
2020-04-29 10:21:58.016189 [ALERT] mod_verto.c:622 WRITE RETURNED ERROR -1 
2020-04-29 10:21:58.016189 [ALERT] mod_verto.c:622 WRITE RETURNED ERROR -1 
2020-04-29 10:21:58.016189 [ALERT] mod_verto.c:622 WRITE RETURNED ERROR -1 
2020-04-29 10:21:58.016189 [ALERT] mod_verto.c:622 WRITE RETURNED ERROR -1 
2020-04-29 10:21:58.016189 [ALERT] mod_verto.c:622 WRITE RETURNED ERROR -1 
2020-04-29 10:21:58.016189 [ALERT] mod_verto.c:622 WRITE RETURNED ERROR -1 
2020-04-29 10:21:58.016189 [ALERT] mod_verto.c:622 WRITE RETURNED ERROR -1 
2020-04-29 10:21:58.016189 [ALERT] mod_verto.c:622 WRITE RETURNED ERROR -1 
2020-04-29 10:21:58.016189 [ALERT] mod_verto.c:622 WRITE RETURNED ERROR -1 
2020-04-29 10:21:58.016189 [ALERT] mod_verto.c:622 WRITE RETURNED ERROR -1 
2020-04-29 10:21:58.016189 [ALERT] mod_verto.c:622 WRITE RETURNED ERROR -1 
2020-04-29 10:21:58.016189 [ALERT] mod_verto.c:622 WRITE RETURNED ERROR -1 
2020-04-29 10:21:58.016189 [ALERT] mod_verto.c:622 WRITE RETURNED ERROR -1 
2020-04-29 10:21:58.016189 [ALERT] mod_verto.c:622 WRITE RETURNED ERROR -1
(...)

and after that i cannot connect to verto. The only solution is to reload mod_verto. 

For what i could see, the -1,  is because in function ws_write_frame, wsh->down is true.
This should affect only a single client or it affects all of them?
How can avoid this loop?

Thanks,

**Comment by wmasilva** [comment]
i've lot of clients that i  think they are zombies:
       lan::@	client	                       176.58.10.164:62409	CONN_NO_REG (WSS)
       lan::@	client	                       176.58.10.164:57269	CONN_NO_REG (WSS)
       lan::@	client	                       176.58.10.164:56717	CONN_NO_REG (WSS)
      lan::@	client	                       176.58.10.164:58371	CONN_NO_REG (WSS)

Then in the verto client i see lot of error messages that match the -1 alert message:
[2020-05-13 12:48:02.988] [error] Websocket Lost 0 sleep: 1000msec
[2020-05-13 12:48:03.993] [info] Attempting Reconnection....
[2020-05-13 12:48:04.166] [info] logged in

Any tip to see what could be causing the issue?

**Comment by nerdbaggy** [comment]
Yup having the same issue. Are you disconnecting from the socket when leaving the page?

**Comment by sloweclair** [comment]
What verto library are you using? Are you using the standard jquery one?

**Comment by anthmFS** [comment]
not sure if this is your problem but: if you are using a self-signed cert you have to visit it at least once via https:// to agree to the exception because there is no dialog for that on WebSockets wss://

**Comment by wmasilva** [comment]
yes, using the standard verto.js. If i don't accept the cert exception it doesn't connect.  The issue happens after sometime connected.
